### ISSUE TITLE
intenity information

### ISSUE THREAD

**Issue by HaroldMurcia** [QuanergySystems/quanergy_client_ros#26]
Hello,
Reviewing the data generated in a test with the Quanergy M8 ultra the question arises:
* Is the intensity response natural, or does it have some kind of normalisation (distance, angle) or processing? 
* Do the echoes affect the intensity response?

Regards,

**Comment by rossctaylor** [comment]
The intensity is range normalized and reported on a logarithmic scale of return energy vs transmit energy. Thus, it is primarily a measure of reflectivity but can be affected by other factors that might reduce the return energy. Multiple echoes would imply the energy is split between multiple surfaces which would reduce the energy return for each echo and thus the intensity.

**Comment by HaroldMurcia** [comment]
Thank you for your quick response. Some papers suggest range and angle normalizations in intensity:

$I_{norm} = i * R^2 / cos(\alpha) *K$

where **i** is the original intenisty, **R** is the range [m], **$\alpha$** the incidence angle and **K** a tuning constant.  

- According to your previous answer, does it mean that the Quanergy M8 already contains (inside) a range correction? 
- Is it possible to know the equation applied on the original intensity?

**Comment by rossctaylor** [comment]
Yes, range correction is built in. We've elected not to publish the function applied.
